packages/cart/src/migrations/CartModuleSetup20240122122952.ts

```diff
@@ -0,0 +1,98 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class CartModuleSetup20240122122952 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'create table "cart_address" ("id" text not null, "customer_id" text null, "company" text null, "first_name" text null, "last_name" text null, "address_1" text null, "address_2" text null, "city" text null, "country_code" text null, "province" text null, "postal_code" text null, "phone" text null, "metadata" jsonb null, "created_at" timestamptz not null default now(), "updated_at" timestamptz not null default now(), constraint "cart_address_pkey" primary key ("id"));'
```

Comment:
I suggest to add `if not exists` everywhere. Creating tables, constraints and indexes.
That will avoid issues when modularizing things if you want to move it to another DB for example, where you copy all the data, but you might not have the migrations of that module.